backend/ontario/ontario/__init__.py

```diff
@@ -393,6 +393,32 @@ def tileglass(self, width: int, height: int) -> "ImageBuilder":
         self.__nodes.append(node)
         return self
 
+    def text(self,
+             string: str,
+             font: str,
+             size: float,
+             color: tuple,
+             wrap_width: int,
+             wrap_height: int,
+             alignment: int,
+             vert_alignment: int) -> "ImageBuilder":
+        """
+        Produce text pixel buffer
+        """
+
+        node = self.__parent.create_child("gegl:text")
+        node.set_property("string", string)
+        node.set_property("font", font)
+        node.set_property("size", size)
+        node.set_property("color", Gegl.Color.new(color))
+        node.set_property("wrap_width", wrap_width)
+        node.set_property("wrap_height", wrap_height)
+        node.set_property("alignment", alignment)
+        node.set_property("vert_alignment", vert_alignment)
+
+        self.__nodes.append(node)
+        return node.get_property("width"), node.get_property("height")
```

Comment:
Is this an ImageBuilder?

Reply:
This generates text as an image buffer and returns the width and height of the theoretical text so you can construct a shadow buffer with that information later.

Reply:
The type information here says that it returns an `ImageBuilder` though...

https://github.com/TeamPictonode/pictonode/blob/2452364a03b6c22abe91ee7136877393fd0af5d5/backend/ontario/ontario/__init__.py#L404

Reply:
I see, that was just copied and pasted, will fix. Thanks!